cookbook/form/data_transformers.rst

```diff
@@ -16,24 +16,24 @@ to render the form, and then back into a ``DateTime`` object on submit.
     When a form field has the ``inherit_data`` option set, Data Transformers
     won't be applied to that field.
 
-Simple Example: Sanitizing HTML on User Input
----------------------------------------------
+Simple Example: Transforming tags string from User Input to array
```

Comment:
According to our headline standards this should be "Simple Example: Transforming String Tags from User Input to an Array".